plugins/modules/k8s_scale.py

```diff
@@ -194,7 +202,10 @@ def execute_module(
     if wait:
         return_attributes["duration"] = 0
 
-    resource = k8s_ansible_mixin.find_resource(kind, api_version, fail=True)
+    svc = K8sService(client, module)
+    resource = svc.find_resource(
+        module.params.get("kind"), module.params.get("api_version"), fail=True
+    )
 
     from ansible_collections.kubernetes.core.plugins.module_utils.common import (
         NotFoundError,
```

Comment:
This should be removed and we should import this directly from kubernetes so we remove the dependency on common.py